### DISCUSSION TITLE
I win a block and the reward do not arrive

### DISCUSSION THREAD

**Discussion by dejangaric85** [Chia-Network/chia-blockchain#8584] [Support]
Greeting,

I won a block at a space pool but didn’t get a prize on my wallet.

I won blok #894249 in space pool and there is no revenue yet
9/22/21, 10:09 AM

Hash is 9c98636dabed7f380f2f1a1b53736b3d9f5cbd37c1498cc2903ec47379654eeb

I checked my award at https://xchscan.com/

My farmer's address for the prize is the one that did not receive the prize from the block won :
           xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5

The farmer's address to which I receive rewards from the space pool is : 
          xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja

Please help me with this problem, thanks a lot in advance.

I will send you in addition what it takes from the data to solve this problem.

Best regards,
Dejan

**Comment by dejangaric85** [comment]
My first wallet address is  :   
xch1ykh4nrv8ytcvugjyeupfcq7kw4t22g0khxzs4jrdqawvtldj3f7q6jd70t

**Comment by dejangaric85** [comment]
If anyone can help, there is no response from support?

**Comment by loppefaaret** [comment]
if you tell me what the address is under the "Farm" tab, under the 3 `...` menu in the top right, the "farmer rewards target address" ?
it is likely set to `xch1ykh4nrv8ytcvugjyeupfcq7kw4t22g0khxzs4jrdqawvtldj3f7q6jd70t` - by default, this is the address of your first wallet, its first wallet address
if that is the case, you did not win block 894249 - that was won by a farmer that had the address `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` in its "farmer rewards target address"

if you use CLI, the "farmer rewards target address" can be found as the address under the `xch_target_address:` under the `farmer:` section in `config.yaml`

please note that i have had this question multiple times from pool.space farmers, and all of them were infact not the winners of the block that space pool said they won!

**Reply by siteregister** [reply]
> if you tell me what the address is under the "Farm" tab, under the 3 `...` menu in the top right, the "farmer rewards target address" ? it is likely set to `xch1ykh4nrv8ytcvugjyeupfcq7kw4t22g0khxzs4jrdqawvtldj3f7q6jd70t` - by default, this is the address of your first wallet, its first wallet address if that is the case, you did not win block 894249 - that was won by a farmer that had the address `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` in its "farmer rewards target address"
> 
> if you use CLI, the "farmer rewards target address" can be found as the address under the `xch_target_address:` under the `farmer:` section in `config.yaml`
> 
> please note that i have had this question multiple times from pool.space farmers, and all of them were infact not the winners of the block that space pool said they won!

Hi loppefaaret,
I have same problem yok but my address under the "Farm" tab, under the 3 `...` and on link to say you win a block ( https://xchscan.com/blocks/2282412 ) iş same and no any rewards xch.
Can you help me please?

**Reply by loppefaaret** [reply]
if you go to your farming tab, and open up the 3`...` dot menu, and check the farming / pool rewards target addresses - do you also see a warning about the addresses not being controlled by your wallet ?

**Reply by siteregister** [reply]
![16595086124271562420511522857335](https://user-images.githubusercontent.com/109757300/182540673-1d15b60e-33bc-40b5-a011-85a616056bd4.jpg)
I think you mean this warning?

**Reply by loppefaaret** [reply]
yes, the rewards are being sent to an address belonging to a wallet _not_ on the farming machine - if you dont recognizing specifically doing this, take a receive address from you wallet, and input as the rewards target address

**Reply by siteregister** [reply]
Sorry i didnt understand. Can you description how i do ? And i am so sorry for my bad English.

**Comment by dejangaric85** [comment]
my address in farming at 3 points in the corner is : 
xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja

my address in config yaml is xch_target_address: 
xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja

**Reply by loppefaaret** [reply]
yup, then you did not win block https://xchscan.com/blocks/894249 sorry

**Reply by dejangaric85** [reply]
as I did not win the block, it says on the space pool that my address opened the block, and on their website it says that I won the block but there is no prize.

my first address on my wallet is : 
xch1ykh4nrv8ytcvugjyeupfcq7kw4t22g0khxzs4jrdqawvtldj3f7q6jd70t

but the farmer has set the address to which I received all the awards from the space pool, I changed it when my address changed in the wallet and I put this address which says that she won the block but I did not receive the payment at that address : 
xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja

**Comment by loppefaaret** [comment]
as long as your "farmer rewards target address" is pointing to a receive address from a wallet / key that you control, that is good - which address does not really matter, they all is controlled by the same wallet / key...
but i'm afraid that neither of the above addresses has been credited with a win on the blockchain!

**Reply by dejangaric85** [reply]
this all it says are my addresses, I checked it first, I wouldn't have written to support if I hadn't checked it first on this site



Block Height
894,249
Header Hash
0x0xe3595a5c653b44e9749e4605edba3e21ee9e62dc29e09b0a09b83af12ea2204d
Timestamp
N/A
Transactions
N/A
Farmer
xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5
Farmer Puzzlehash
0x018e25fb70bd2af22ca0446f0f167eb6a682dbc8fcf9487390aa9bf57c6588f8
Pool
xch1ferwzm93dlkp8lsk9hg3u6rrgc7mw6ah8qpw93nlgdyalxrg6y0sytfpxf
Pool Puzzlehash
0x4e46e16cb16fec13fe162dd11e6863463db76bb73802e2c67f4349df9868d11f
Pool Reward
1.75 XCH
Farmer Reward
0.25 XCH
Fees

this is with the space pool what it says when i won the block and it is also all my addresses that are set on my wallet



You have found 1 block

According to the Chia pooling protocol, 0.25 XCH is given directly to the farmer who finds a block. This reward does not go through Space Pool. If you can't find it, refer to this post.

Date
9/22/21, 10:09 AM
Block #
894249
Hash
9c98636dabed7f380f2f1a1b53736b3d9f5cbd37c1498cc2903ec47379654eeb


i don't know what happened so i didn't get that award that was supposed to be on my wallet

**Reply by loppefaaret** [reply]
try putting `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` as your "farmer rewards target address" - if the GUI shows a warning message about not controlling the address, it is clearly not your address, and not you that won it, however much you, space pool, or any one else but the Chia Blockchain data is telling you, you won!
I am sorry that you did not win it, but it really is a matter of someone telling you something that isnt true...
apart from you being able to control the address `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` - you did not win that block - or you having `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` previously set as your "farmer rewards target address" while the block number above was farmed...

**Reply by dejangaric85** [reply]
that address is also set up for me in manage farming reward



Manage Your Farming Rewards Target Addresses
Farmer Reward Address
xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja
Pool Reward Address
xch1atrql06qensep8rgpdgk8s5aes06xykln3wjevg8z8jy0euughcszzlvja
Note that this does not change your pooling payout addresses. This only affects old format plots, and the 0.25XCH reward for pooling plots.


I copied what was set

**Reply by dejangaric85** [reply]
I control all those addresses, they are all my addresses but the prize did not arrive and it says that it came to my address that was set

**Reply by loppefaaret** [reply]
so inputting `xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5` as the `farmer rewards target address` does not show a warning about not controlling the address ?

**Reply by dejangaric85** [reply]
I put this address in the farmer rewards target and here's what it says and how, I didn't get a warning and I don't know what it says above above if you can tell me





Manage Your Farming Rewards Target Addresses
No private keys for one or both addresses. Safe only if you are sending rewards to another wallet.
Farmer Reward Address
xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5
Pool Reward Address
xch1qx8zt7msh540yt9qg3hs79n7k6ng9k7glnu5suus42dl2lr93ruqz2slz5
Note that this does not change your pooling payout addresses. This only affects old format plots, and the 0.25XCH reward for pooling plots.

**Reply by emlowe** [reply]
This `No private keys for one or both addresses. Safe only if you are sending rewards to another wallet` is the aforementioned warning we are talking about. You don't have the key for that address

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?

**Comment by mouracomp2** [comment]
I come through this to obtain information, from blocks collected in SpacePool, because my percentage in my wallet is not paid.
my Wallet 280308653
my payment address was
xch1u4tycqne2aqrgcvvfnk2sjqlgwynucqggv6j4kg97wps3hkj4wds5kn73p
Date
08/08/2022 00:21
07/23/2022 02:07
04/25/2022 06:25
03/22/2022 14:01
12/29/2021 20:12
block #
2369220
2295897
1886491
1730969
1349021
hash
08795929d431ec810371dfe48e609eb1d1e395a96be78186627c178ad6d9b742

6896a896874b87b5be2743ce61a53c0ea3ce759c30b81c393eed25f9149b1af5

57304513456528bd42bcf36785ff47ba9670c971dc949658def3e2a610ef3fa0

ff2f214133248308b1b4459dd9f296da1a7430554919185713e2b3b9522cf2ee

29c941f2d9dfe4a4e687bbc61119033e5249f227d7a994d4f91533f7b174f420

never been paid, any explanation, what am i doing wrong?
